Improving and changing images from the Tracks sections (#600)
opengisch/QField-docs: documentation/how-to/tracking.en.md

# Tracking

:material-tablet: Fieldwork

> QField allows you to create points, lines or polygons and track yourself in the background while working on another feature layer or while having the device in your pocket. To enable tracking, you need to ensure that your positioning is activated.

Reviewer: ```suggestion
QField allows you to track your position through the creation of points, lines or polygons while browsing the map, working on other features and layers, or in the background while having the device in your pocket. To enable tracking, you need to ensure that your positioning is activated.
```

## Set-Up Tracking Session

> 1. Open the legend and press long on the layer which you want to use for tracking
> 2. Select **Setup tracking** and a new window will appear showing the configuration settings

Reviewer: @beanzmo , we should come up with a choice on whether we want to use paragraphs or bullet points, I feel ATM we're just hoping from one style to another and that's providing for a sense of inconsistency.

```suggestion
1. Open the dashboard and long press on the legend layer which you want to use for tracking
2. Select **Setup tracking** and a new popup will appear showing the configuration settings
```

    !![](../assets/images/tracking-layer-properties.png,350px)

    You can choose between two options how the number of vertices will be recorded during the tracking session:

    - The `Time` tolerance records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
    - The `Distance` tolerance is more efficient, recording a point only after you move a minimum distance.
    This keeps your track clean by ignoring movement when you are stationary.
    To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.

    !![](../assets/images/tracking-settings.png,350px)

>     In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.

Reviewer: ```suggestion
    The erroneous distance safeguard option can be turned on to insure sporadic bad GNSS readings are not tracked. The functionality relies on providing a maximum tolerated distance from the last recorded position beyond which readings will be skipped.
```

    !![](../assets/images/maximum-distance-tolerance.png,350px)

3. Tap the **Start tracking** button to begin the tracking.

> 4. Set the attributes for the feature(s) about to be created.

Reviewer: See here the bullet point presentation definitively feels off, it's intertwined with extra paragraphs and images.

!!! note
    For line and polygon layers, a single feature will be created per tracking session, with its geometry formed from the recorded positions.
    For point layers, a new feature will be created for each recorded position, and the attribute values entered in the form will be remembered for subsequent points.

!!! tip
    QField will skip the feature form if the layer is configured to hide all attributes or if the form has been set to hide on feature addition in the attribute form settings.
    This can streamline the process of starting tracking sessions.

## Resuming to a Previous Session

> If did not explicitly terminate your tracking session (for example, you closed the app), QField allows you to continue where you left off.

Reviewer: ```suggestion
QField allows you to continue a track where you left off.
```

> A prompt will appear asking whether to **resume the last session** or **start a new one**.

Reviewer: ```suggestion
When setting up a tracking session, simply select the **resume the last session** to continue the last tracking geometry.
```

- **Resuming** a session for a line or polygon layer will continue adding vertices to the feature from the previous session.
This allows you to simply continue across app restarts.

> - **Starting a new session** will discard the incomplete feature from the previous session and begin a new one.

Reviewer: There again we jump from paragraphs vs bullet point (in this case to say the same thing as what we said in the paragraph above :) )

## Tracking underway

Once a tracking session has been set up, a badge will appear in the side dashboard legend next to the layer(s) against which tracking is being recorded.

!![](../assets/images/tracking-badge.png)

The features created by the tracking session are saved on every vertex recorded.
A rubberband line overlayed onto the map is attached to each tracking session, allowing you to glance at what has been recorded.

If the layer linked to a tracking session supports the M dimension, QField will store
the time passed since the first vertex recorded in each vertex's M value.

!!! tip
    You can simultanously record multiple trackings sessions across several layers.

## Terminate tracking

> 1. Open the side dashboard and press long on the tracking layers layer

Reviewer: ```suggestion
1. Open the side dashboard and press on the layer's tracking badge you want to stop.
```

> 2. Select the *Stop tracking* button.

Reviewer: ```suggestion
2. Push the *Stop* button within the toast message that appears when you clicked on the tracking badge.
```

!![](../assets/images/tracking-stop.png)

> ## Automatic Tracking Sessions

Reviewer: I think project tracing sessions was a better term here, why changing?

This functionality enables one or more position tracking sessions to automatically start upon project load.
The feature form of the layer will immediately open asking you for the attributes.
If the tracked vector layer has the "Hide Form on Add Feature" mode selected, the feature form will be skipped.

!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)

> ### Configuration of an automatic Tracking Session

Reviewer: See above about project tracking session being IMHO a better name. Also, title case is not respected.

:material-monitor: Desktop preparation

> 1. Open the properties of the your tracking layer and direct to the QField tab .

Reviewer: ```suggestion
1. Open the properties of the vector layer you want to track your position against and go to the QField tab.
```

# Tracking

:material-tablet: Fieldwork

QField allows you to track your position through the creation of points, lines or polygons while browsing the map, working on other features and layers, or in the background while having the device in your pocket.

> To enable tracking, you need to ensure that your positioning is activated.

Reviewer 2: ```suggestion
To enable tracking, you need to ensure that positioning is activated.
```

## Set-Up Tracking Session

> 1. Open the main dashboard legend and press long on the layer which you want to use for tracking.

Reviewer 2: ```suggestion
1. Open the main dashboard legend and long-press on the layer which you want to use for tracking.
```

2. Select **Setup tracking** and a new window will appear showing the configuration settings.

    !![](../assets/images/tracking-layer-properties.png,350px)

    You can choose between two options how the number of vertices will be recorded during the tracking session:

    - The `Time requirement` records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
    - The `Distance requirement` is more efficient, recording a point only after you move a minimum distance.

>     This keeps your track clean by ignoring movement when you are stationary.

Reviewer 2: ```suggestion
      This keeps your track clean by ignoring movement when you are stationary.
```

This is part of the distance I assume?

    To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.

    !![](../assets/images/tracking-settings.png,350px)
    - The erroneous distance safeguard option can be turned on to insure sporadic bad GNSS readings are not tracked.
    The functionality relies on providing a maximum tolerated distance from the last recorded position beyond which readings will be skipped.

>     In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.

Reviewer 2: ```suggestion
    To prevent the next vertex from being set too far from the most recent one (e.g., in steep terrain where elevation matters), set a maximum tolerance distance.
```

Shorter, easier to read

!!! tip
    QField will skip the feature form if the layer is configured to hide all attributes or if the form has been set to hide on feature addition in the attribute form settings.
    This can streamline the process of starting tracking sessions.

## Resuming to a Previous Session

If did not explicitly terminate your tracking session (for example, you closed the app), QField allows you to continue where you left off.
A prompt will appear asking whether to **resume the last session** or **start a new one**.

- **Resuming** a session for a line or polygon layer will continue adding vertices to the feature from the previous session.
This allows you to simply continue across app restarts.

- **Starting a new session** will discard the incomplete feature from the previous session and begin a new one.

## Tracking underway

Once a tracking session has been set up, a badge will appear in the side dashboard legend next to the layer(s) against which tracking is being recorded.

!![](../assets/images/tracking-badge.png)

The features created by the tracking session are saved on every vertex recorded.

> A rubberband line overlayed onto the map is attached to each tracking session, allowing you to glance at what has been recorded.

Reviewer 2: ```suggestion
During a tracking session, a rubberband line appears on the map to visualize the recorded path.
```

# Tracking

:material-tablet: Fieldwork

QField allows you to create points, lines or polygons and track yourself in the background while working on another feature layer or while having the device in your pocket. To enable tracking, you need to ensure that your positioning is activated.

## Set-Up Tracking Session

1. Open the legend and press long on the layer which you want to use for tracking
2. Select **Setup tracking** and a new window will appear showing the configuration settings

!![](../assets/images/tracking-layer-properties.png,350px)

You can choose between two options how the number of vertices will be recorded during the tracking session:

- A minimum time interval
- A minimum distance

    !![](../assets/images/tracking-settings.png,350px)

> In the case that you wish to ensure that the next vertex is set not too far away from the most recent one (for example if you are in steep terrain and elevation is relevant), you can set a maximum tolerance distance.

Reviewer 2: "distance" tolerance?

Author: Thanks for the suggestions, I rephrase it a bit in the last commit.

    You can choose between two options how the number of vertices will be recorded during the tracking session:

    - The `Time` tolerance records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
    - The `Distance` tolerance is more efficient, recording a point only after you move a minimum distance.
    This keeps your track clean by ignoring movement when you are stationary.

>     To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.

Reviewer 2: In the user interface this is called "Time requirement" and "Distance requirement". @nirvn might be convinced that tolerance is a better fit in terms of wording (@nirvn ?) but we should use a consistent terminology.

Reviewer: I'm not convinced :wink:  This is not a tolerance, it is a requirement as indicated by both QFieldSync and QField.

<img width="585" height="221" alt="image" src="https://github.com/user-attachments/assets/69be1891-36a7-426a-a460-74e4153ec292" />

<img width="585" height="221" alt="image" src="https://github.com/user-attachments/assets/c8819c41-6f7a-43a4-b097-ebd2ce84fbbe" />

A tolerance indicates a "flexible" behavior, i.e. I'd read a time tolerance as "add at any time between 0 to 30 sec" :)
